docset/windows/scheduledtasks/new-scheduledtasktrigger.md

```diff
@@ -216,7 +216,7 @@ Accept wildcard characters: False
 ```
 
 ### -CimSession
-Specifies Common Information Model (CIM) session object that represents a connection to a local computer or a remote computer.
+Runs the cmdlet in a remote session or on a remote computer. Enter a computer name or a Common Information Model (CIM) session object that represents a connection to a local computer or a remote computer, such as the output of a New-CimSession or Get-CimSession cmdlet. The default is the current session on the local computer.
```

Comment:
The default is the current session on **a** local computer.

Reply:
Ok I think that "the local computer" is needed here instead of "a local computer" because we need to make sure this is understood to be run on the computer that the user is currently running the cmdlet and not "any local computer", for that option you'll need to use remote computer instead.
Thank you

Reply:
thanks, copy edits should never be allowed to change technical meaning. appreciate your response.